Restart npm audit

<!--- Provide a general summary of the issue in the Title above -->

## Description
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug. What were you hoping to achieve? -->
We should restart `npm audit` in our builds when we get the following occur:

```
npm audit
npm ERR! code ENOAUDIT
npm ERR! audit Your configured registry (https://registry.npmjs.org/) does not support audit requests.
```
## Expected Behavior
<!--- Tell us what should happen -->

## Actual Behavior
<!--- Tell us what happens instead -->

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- Reproduce this bug include code to reproduce, if relevant -->
1.
2.
3.

## Existing issues
<!-- Have you searched for any existing issues or are their any similar issues that you've found? -->
- [ ] [Stack Overflow issues](https://stackoverflow.com/questions/tagged/ibp-vscode-extension)
- [ ] [GitHub Issues](https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues)

<!-- Please include any links to issues here -->

## Log File

<!-- In Visual Studio Code's command palette, run '>Developer: Open Logs Folder' -->
<!-- Attach/link to the latest nested 'Blockchain.log' file -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* IBM Blockchain Platform Extension Version:
* Visual Studio Code version:
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
* Operating System and version:
* Link to your project:

--- a/azure-templates/setup-steps.yml
+++ b/azure-templates/setup-steps.yml
@@ -28,8 +28,12 @@ steps:
       echo ">>> Compile vscode-test"
       npm install --no-optional
       npm install --ignore-scripts
-      n=0; rc=256;
+    displayName: run npm install
+
+  - script: |
+      set -ev
       function runNpmAudit() {
+        local n=0; local rc=256; local out=""
         while [ "$n" -ne  10 ] && [ "$rc" -ne 0 ]; do
           echo ">>> run npm audit - Attempt $n"
           out=$((npm audit) 2>&1)
@@ -42,8 +46,13 @@ steps:
         done
         return $rc;}
       runNpmAudit
+      exit $?
+    displayName: run npm audit
+
+  - bash: |
+      set -ev
       npm run compile
       echo ">>> Compiled vscode-test"
-    displayName: Install stuff
+    displayName: run npm compile
     env:
       DISPLAY: ':99.0'

